README.md

```diff
@@ -28,8 +28,25 @@ An example of the `BrowserCommunicationTypes.js` is saved in [`examples/BrowserC
 The only important function is basically `BrowserCommunication.addListener`. 
 Just pass to it a set-up [message type](#message-types) and a callback to actually register. For the former, it is obviously recommend to use the constant from there and pass it into the function.
 
-You can register as many listeners for one message type, as you want.
+You can register as many listeners for one message type, as you want. Example:
+
+```js
+const firstCallback = () => console.log('First callback called!!');
```

Comment:
while that may technically work, actually the example should demonstrate the proper way without inline "magical" strings and use the `COMMUNICATION_MESSAGE_TYPE` type/object for defining the message ID.